crates/accelerate/src/sampled_exp_val.rs

```diff
@@ -53,6 +67,41 @@ fn bitstring_expval(dist: &HashMap<String, f64>, mut oper_str: String) -> f64 {
     exp_val / denom
 }
 
+fn bitstring_expval_bitterm(
+    dist: &HashMap<String, f64>,
+    term: &SparseTermView,
+) -> Result<f64, PyErr> {
```

Comment:
We typically just use `PyResult` here, which is a shorthand notation for `Result<T, PyErr>` 🙂 
```suggestion
) -> PyResult<f64> {
```
Below you're using `PyResult<f64>` sometimes, but it's a bit inconsistent at the moment

Reply:
Changed all to `PyResult`.